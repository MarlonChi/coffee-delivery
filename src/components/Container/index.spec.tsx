import { renderWithTheme } from "../../utils/tests/renderWithTheme";

import { Container } from ".";

describe("<Container/>", () => {
  it("should render container", () => {
    const { container } = renderWithTheme(
      <Container>
        <span>Coffe Delivery</span>
      </Container>
    );

    expect(container.firstChild).toHaveStyleRule("max-width", "144rem");

    expect(container.firstChild).toMatchSnapshot();
  });
});
