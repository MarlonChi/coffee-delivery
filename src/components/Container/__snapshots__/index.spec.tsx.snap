// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container/> should render the heading 1`] = `
.c0 {
  width: 100%;
  max-width: 144rem;
  margin-left: auto;
  margin-right: auto;
}

<div
  class="c0"
>
  <span>
    Coffe Delivery
  </span>
</div>
`;
